neo_force_scheme/pca.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+import time
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import seaborn as sns
+import sys
+from matplotlib.colors import ListedColormap
+from sklearn import datasets
+
+def excute_pca(dataset):
+    print("Importing data.")
+    rawdata = dataset
+    print("Done.")
```

Comment:
its better not to print anything here or below (there are other prints) unless you use the "verbose" parameter to check if the user wants a verbose way of printing. Check how I do that in the neo_force_scheme file.